frontend/src/js/components/settings/organization/Organization.tsx

```diff
@@ -187,7 +187,7 @@ export const Organization = () => {
                 dividerDisabled
                 key="org_token"
                 onExpansion={onTokenExpansion}
-                secondary={org.tenant_token}
+                secondary={showTokenWarning ? tenant_token : `${tenant_token.substring(0, 5)}...${tenant_token.substring(tenant_token.length - 5)}`}
```

Comment:
IMO if we use asterisks instead of dots, we'll signify that it's a sensitive information, not just information that didn't fit in the container